content/docs/sidebar.json

```diff
@@ -101,6 +101,7 @@
         "source": false,
         "children": ["undo-adding-data", "update-tracked-files"]
       },
+      "setup-dropbox-remote",
       "setup-google-drive-remote",
```

Comment:
We prob want GDrive first?

Reply:
I guess, it doesn't matter ... any specific reasons for this @jorgeorpinel that you have in mind?